helm/grafana-multi-tenant-proxy/templates/config.yaml

```
@@ -0,0 +1,11 @@
+{{- if .Values.proxy.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "proxy.fullname" $ }}-config
```

We might exceed the 63 characters limit here

---

Fixed with https://github.com/giantswarm/grafana-multi-tenant-proxy/pull/152/files#diff-1e08bd35de07099af400a8bdae61da19bb99b93ba0b9c126f89a9eea94d49648R16

---

Can we instead use this ?
```suggestion
  name: {{ include "proxy.fullname" $ }}
```